CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.0.0)
+project(calculator VERSION 0.1.0)
+
+include(CTest)
+enable_testing()
+
+include_directories(stack)
```

Comment:
why is stack directory needed here? and why is cmake being used if make file is intact?